docs/advanced/more/migrate_adapter_zigpy.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar: auto
+---
+
+# Migrate to another adapter with zigpy
+
+::: warning
+Migration from one adapter to another requires backup and restore support which is so far only implemented for the `zstack` (Texas Instrument) and `ember` adapters in Z2M.
+Backup and restore is **not officially supported** for any other adapters (`conbee`, `ezsp`, `zboss` and `zigate`).
+Please don't open issues about this method, but feel free to discuss it in the discussions board.
```

Comment:
> but feel free to discuss it in the discussions board.

Maybe we can create a discussion already and link to that?

Reply:
Just done that. https://github.com/Koenkk/zigbee2mqtt/discussions/26716

Reply:
Sorry for switching opinions, I was discussing this with @Nerivec and we also understood from @puddly that results may vary (and might not even work in some cases). Maybe it's best to just put the markdown page in your discussions now (instead of the official docs), then we can gather some feedback and iterate from there. We can leave the link to the discussion on the faq page.

Reply:
Conbees with firmware 0x26720700 or newer, EmberZNet, and Z-Stack all can be migrated to back and forth without issues. For EmberZNet you need to be running I believe 7.4.0.0 or newer to have the IEEE address migrated. `zigpy-cli` should warn you about all of these when you run it.

All other adapters don't support migration in any meaningful way (ZiGate can't set the PAN ID or export the network key and we don't really support ZBOSS yet).

Reply:
Thank you @puddly for giving us more insight. I have moved the guide to the discussion board and removed it from the main website doc.